packages/mwp-app-server/src/index.js

```diff
@@ -37,25 +41,30 @@ export default function start(languageRenderers, { routes = [], plugins = [] })
 	const baseRoutes = getRoutes();
 	const finalRoutes = [...routes, ...baseRoutes];
 
-	const connection = {
+	const serverConfig = {
 		host: '0.0.0.0',
 		port: appConfig.app_server.port,
-		routes: {
-			plugins: {
-				'electrode-csrf-jwt': {
-					enabled: false,
-				},
-				'mwp-logger-plugin': {
-					enabled: true,
-				},
+
+		// accessed via server.settings.app
+		app: appConfig,
+
+		// accessed via server.settings.plugins
+		plugins: {
+			'electrode-csrf-jwt': {
+				enabled: false,
```

Comment:
@mmcgahan Do you know why we set this to `false` at this point?

Reply:
so that CSRF checking is _off_ by default, and individual routes can opt-in, rather than it being an implicit behavior that routes have to turn _off_